rca/enquire_to_study/views.py

```diff
@@ -233,7 +240,8 @@ def delete(request):
         request,
         "enquire_to_study/confirm_delete.html",
         {
-            "count": count,
+            "count_all_submissions": EnquiryFormSubmission.objects.count(),
```

Comment:
I think instead of making 2 queries for `EnquiryFormSubmission.objects` we could do the following

```
count_submissions = EnquiryFormSubmission.objects.all()
instances = count_submissions.objects.filter(
        Q(submission_date__lte=time_threshold)
    )
```

Then use those vars in the context method?